.mailmap

```diff
@@ -829,6 +829,8 @@ Lukas Zorich <lukas.zorich@gmail.com>
 Luke Peterson <hazelnusse@gmail.com>
 Luv Agarwal <agarwal.iiit@gmail.com>
 Maciej Baranski <getrox.sc@gmail.com>
+Maciej Skorski <31315784+maciejskorski@users.noreply.github.com>
+Maciej Skorski <maciej.skorski@gmail.com> maciejskorski <maciej.skorski@gmail.com>
```

Comment:
Both lines should begin with the name/email that you want to have in the AUTHORS file:
https://github.com/sympy/sympy/blob/45572b52a401d73fa17001a28ee28276d6e76b1c/.mailmap#L138-L148